Translations update from Kodi Weblate

weblate opened the pull request:
Translations update from [Kodi Weblate](https://kodi.weblate.cloud) for [Kodi add-ons: PVR clients/pvr.octonet](https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/).



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widget/kodi-add-ons-pvr-clients/pvr-octonet/horizontal-auto.svg)

garbear commented:
Translations have now been updated to use the Piers branch. Feel free to merge translation PRs as often as you'd like.

garbear commented:
This PR can be merged for the latest translations.

garbear commented:
@DigitalDevices Can you merge?

weblate committed:
Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 100.0% (12 of 12 strings)

Deleted translation using Weblate (Ossetian (os_os))

Deleted translation using Weblate (Kannada (India) (kn_in))

Deleted translation using Weblate (Occitan (France) (oc_fr))

Deleted translation using Weblate (Occidental (ie_ga))

Deleted translation using Weblate (Irish (ga_ie))

Deleted translation using Weblate (Filipino)

Deleted translation using Weblate (Irish)

Deleted translation using Weblate (Hebrew)

Deleted translation using Weblate (English (United Kingdom))

Added translation using Weblate (Hebrew)

Added translation using Weblate (Irish)

Added translation using Weblate (English (United Kingdom))

Added translation using Weblate (Filipino)

Added translation using Weblate (Irish (ga_ie))

Added translation using Weblate (Occidental (ie_GA))

Added translation using Weblate (Ossetian (os_os))

Added translation using Weblate (Kannada (India) (kn_in))

Added translation using Weblate (Occitan (France) (oc_fr))

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Bosnian (Bosnia and Herzegovina) (bs_ba))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (12 of 12 strings)

Added translation using Weblate (Dari)

Translated using Weblate (Swedish (sv_se))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Dari (fa_AF))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Persian (Iran) (fa_ir))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Burmese (my_mm))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Vietnamese (vi_vn))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Uzbek (uz_uz))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Turkish (tr_tr))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Thai (th_th))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Tajik (tg_tj))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Telugu (India) (te_in))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Tamil (India) (ta_in))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Silesian)

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Silesian)

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Swedish (sv_se))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Swedish (sv_se))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Serbian (Latin script))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Serbian (Latin script))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Serbian (sr_rs))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Serbian (sr_rs))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Albanian (sq_al))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Slovenian (sl_si))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Slovenian (sl_si))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Sinhala (Sri Lanka) (si_lk))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Portuguese (Portugal) (pt_pt))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Norwegian Bokmål (nb_no))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Norwegian Bokmål (nb_no))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Maltese (mt_mt))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Malay (ms_my))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Malay (ms_my))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Mongolian (mn_mn))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Malayalam (India) (ml_in))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Macedonian (mk_mk))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Maori)

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Latvian (lv_lv))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Lithuanian (lt_lt))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Japanese (ja_jp))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Japanese (ja_jp))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Icelandic (is_is))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Indonesian (id_id))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Indonesian (id_id))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Armenian (hy_am))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Hungarian (hu_hu))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Hungarian (hu_hu))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Hindi (India) (hi_in))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Hebrew (Israel) (he_il))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Galician (Spain) (gl_es))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Galician (Spain) (gl_es))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (French (Canada) (fr_ca))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (French (Canada) (fr_ca))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Faroese (fo_fo))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Basque (Spain) (eu_es))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Basque (Spain) (eu_es))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Spanish (Argentina) (es_ar))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Spanish (Argentina) (es_ar))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Esperanto)

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (English (United States) (en_us))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (English (United States) (en_us))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (English (New Zealand) (en_nz))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (English (New Zealand) (en_nz))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (English (Australia) (en_au))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Greek (el_gr))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Greek (el_gr))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Danish (da_dk))

Currently translated at 75.0% (9 of 12 strings)

Translated using Weblate (Danish (da_dk))

Currently translated at 75.0% (9 of 12 strings)

Translated using Weblate (Welsh (United Kingdom) (cy_gb))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Bosnian (Bosnia and Herzegovina) (bs_ba))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Bulgarian (bg_bg))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Bulgarian (bg_bg))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Belarusian (be_by))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Azerbaijani (az_az))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Asturian (Spain) (ast_es))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Arabic (Saudi Arabia) (ar_sa))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Amharic (Ethiopia) (am_et))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Afrikaans (South Africa) (af_za))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Afrikaans (South Africa) (af_za))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Dari (fa_AF))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Persian (Iran) (fa_ir))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Burmese (my_mm))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Vietnamese (vi_vn))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Uzbek (uz_uz))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Turkish (tr_tr))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Thai (th_th))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Tajik (tg_tj))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Telugu (India) (te_in))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Tamil (India) (ta_in))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Silesian)

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Silesian)

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Swedish (sv_se))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Swedish (sv_se))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Serbian (Latin script))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Serbian (Latin script))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Serbian (sr_rs))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Serbian (sr_rs))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Albanian (sq_al))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Slovenian (sl_si))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Slovenian (sl_si))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Sinhala (Sri Lanka) (si_lk))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Portuguese (Portugal) (pt_pt))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Norwegian Bokmål (nb_no))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Norwegian Bokmål (nb_no))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Maltese (mt_mt))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Malay (ms_my))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Malay (ms_my))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Mongolian (mn_mn))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Malayalam (India) (ml_in))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Macedonian (mk_mk))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Maori)

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Latvian (lv_lv))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Lithuanian (lt_lt))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Japanese (ja_jp))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Japanese (ja_jp))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Icelandic (is_is))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Indonesian (id_id))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Indonesian (id_id))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Armenian (hy_am))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Hungarian (hu_hu))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Hungarian (hu_hu))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Hindi (India) (hi_in))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Hebrew (Israel) (he_il))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Galician (Spain) (gl_es))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Galician (Spain) (gl_es))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (French (Canada) (fr_ca))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (French (Canada) (fr_ca))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Faroese (fo_fo))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Basque (Spain) (eu_es))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Basque (Spain) (eu_es))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Spanish (Argentina) (es_ar))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Spanish (Argentina) (es_ar))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Esperanto)

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (English (United States) (en_us))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (English (United States) (en_us))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (English (New Zealand) (en_nz))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (English (New Zealand) (en_nz))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (English (Australia) (en_au))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Greek (el_gr))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Greek (el_gr))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Danish (da_dk))

Currently translated at 75.0% (9 of 12 strings)

Translated using Weblate (Danish (da_dk))

Currently translated at 75.0% (9 of 12 strings)

Translated using Weblate (Welsh (United Kingdom) (cy_gb))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Bulgarian (bg_bg))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Afrikaans (South Africa) (af_za))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Afrikaans (South Africa) (af_za))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Dari (fa_AF))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Persian (Iran) (fa_ir))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Chinese (Traditional Han script))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Chinese (Simplified Han script))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Burmese (my_mm))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Vietnamese (vi_vn))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Uzbek (uz_uz))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Turkish (tr_tr))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Thai (th_th))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Tajik (tg_tj))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Telugu (India) (te_in))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Tamil (India) (ta_in))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Silesian)

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Silesian)

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Swedish (sv_se))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Swedish (sv_se))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Serbian (Latin script))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Serbian (Latin script))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Serbian (sr_rs))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Serbian (sr_rs))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Albanian (sq_al))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Slovenian (sl_si))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Slovenian (sl_si))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Sinhala (Sri Lanka) (si_lk))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Portuguese (Portugal) (pt_pt))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Norwegian Bokmål (nb_no))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Norwegian Bokmål (nb_no))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Maltese (mt_mt))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Malay (ms_my))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Malay (ms_my))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Mongolian (mn_mn))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Malayalam (India) (ml_in))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Macedonian (mk_mk))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Maori)

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Latvian (lv_lv))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Lithuanian (lt_lt))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Japanese (ja_jp))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Japanese (ja_jp))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Icelandic (is_is))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Indonesian (id_id))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Indonesian (id_id))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Armenian (hy_am))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Hungarian (hu_hu))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Hungarian (hu_hu))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Hindi (India) (hi_in))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Hebrew (Israel) (he_il))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Galician (Spain) (gl_es))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Galician (Spain) (gl_es))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (French (Canada) (fr_ca))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (French (Canada) (fr_ca))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Faroese (fo_fo))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Basque (Spain) (eu_es))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Basque (Spain) (eu_es))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 25.0% (3 of 12 strings)

Translated using Weblate (Spanish (Argentina) (es_ar))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Spanish (Argentina) (es_ar))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Spanish (Spain) (es_es))

Currently translated at 100.0% (12 of 12 strings)

Translated using Weblate (Esperanto)

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (English (United States) (en_us))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (English (United States) (en_us))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (English (New Zealand) (en_nz))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (English (New Zealand) (en_nz))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (English (Australia) (en_au))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Greek (el_gr))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Greek (el_gr))

Currently translated at 8.3% (1 of 12 strings)

Translated using Weblate (Danish (da_dk))

Currently translated at 75.0% (9 of 12 strings)

Translated using Weblate (Danish (da_dk))

Currently translated at 75.0% (9 of 12 strings)

Translated using Weblate (Welsh (United Kingdom) (cy_gb))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 50.0% (6 of 12 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Bosnian (Bosnia and Herzegovina) (bs_ba))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Bulgarian (bg_bg))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Bulgarian (bg_bg))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Belarusian (be_by))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Azerbaijani (az_az))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Asturian (Spain) (ast_es))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Arabic (Saudi Arabia) (ar_sa))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Amharic (Ethiopia) (am_et))

Currently translated at 0.0% (0 of 12 strings)

Translated using Weblate (Afrikaans (South Africa) (af_za))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (Afrikaans (South Africa) (af_za))

Currently translated at 16.6% (2 of 12 strings)

Translated using Weblate (German (de_de))

Currently translated at 100.0% (12 of 12 strings)

Co-authored-by: A B <maxtherabbit85@gmail.com>
Co-authored-by: Alexey <signfinder@gmail.com>
Co-authored-by: Anonymous <noreply@weblate.org>
Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Cutter <CutterXYZ@users.noreply.github.com>
Co-authored-by: Daniel Nylander <daniel@danielnylander.se>
Co-authored-by: Havok Dan <havokdan@yahoo.com.br>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Co-authored-by: José Antonio Alvarado <jalvarado0.eses@gmail.com>
Co-authored-by: Kai Sommerfeld <ksooo@users.noreply.kodi.weblate.cloud>
Co-authored-by: Languages add-on <noreply-addon-languages@weblate.org>
Co-authored-by: Marek Adamski <fevbew@wp.pl>
Co-authored-by: Massimo Pissarello <mapi68@gmail.com>
Co-authored-by: Minho Park <parkmino@gmail.com>
Co-authored-by: Oskari Lavinto <olavinto@protonmail.com>
Co-authored-by: SecularSteve <fairfull.playing@gmail.com>
Co-authored-by: skypichat <skypichat@hotmail.fr>
Co-authored-by: wabisabi926 <liwenliang926@163.com>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/af_za/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/am_et/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/ar_sa/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/ast_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/az_az/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/be_by/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/bg_bg/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/bs_ba/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/ca_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/cs_cz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/cy_gb/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/da_dk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/el_gr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/en_au/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/en_nz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/en_us/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/eo/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/es_ar/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/es_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/es_mx/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/et_ee/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/eu_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/fa_AF/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/fa_ir/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/fo_fo/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/fr_ca/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/fr_fr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/gl_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/he_il/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/hi_in/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/hr_hr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/hu_hu/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/hy_am/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/id_id/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/is_is/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/it_it/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/ja_jp/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/lt_lt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/lv_lv/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/mi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/mk_mk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/ml_in/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/mn_mn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/ms_my/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/mt_mt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/my_mm/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/nb_no/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/nl_nl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/pl_pl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/pt_br/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/pt_pt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/ro_ro/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/si_lk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/sk_sk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/sl_si/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/sq_al/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/sr_Latn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/sr_rs/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/sv_se/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/szl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/ta_in/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/te_in/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/tg_tj/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/th_th/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/tr_tr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/uk_ua/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/uz_uz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/vi_vn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/zh_Hans/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-pvr-clients/pvr-octonet/zh_Hant/
Translation: Kodi add-ons: PVR clients/pvr.octonet

garbear commented:
Translations are ready for merge, if you're able to hit the button.
